pkg/policy/selectorcache.go

```diff
@@ -294,6 +331,12 @@ func (s *selectorManager) setSelections(selections *[]identity.NumericIdentity)
 type fqdnSelector struct {
 	selectorManager
 	selector api.FQDNSelector
+	user     identityPopulator
+}
+
+type identityPopulator interface {
+	StartManagerFQDNSelector(selector api.FQDNSelector) (identities []identity.NumericIdentity, existed bool)
```

Comment:
I find the use of `existed bool` a bit confusing on the call sites, it looks like the idiom on the map lookup, but a value is returned also when `!existed`. Maybe flip the 2nd return value to `isNew bool` instead?